index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="src/style.css">
```

Comment:
Normally, when you write the path of a file, you usually start the path where you are, like this:
```suggestion
	<link rel="stylesheet" href="./src/style.css">
```

Reply:
ok